.vscode/launch.json

```diff
@@ -9,7 +9,9 @@
             // this assumes your workspace is the root of the repo
             "program": "${workspaceFolder}",
             "preLaunchTask": "install",
-            "env": {},
+            "env": {
+                "TF_ACC": "true",
```

Comment:
why should we have acceptance test flag always enabled?

Reply:
So you can run acceptance tests from vscode debugger.  Couldn't enable them otherwise.

Reply:
we can inject t.setEnv("TF_ACC","true") in acceptance test.

Reply:
that would cause it to always be true and take away the protection against not running them by accident when using the cli.  This just causes vscode to launch the debug process with acc enabled.  It doesn't affect the cli.  Could add a second configuration to this file if you wanted another debug target where it was explicitly disabled.